include/odp/api/spec/crypto.h

```diff
@@ -559,7 +559,7 @@ typedef struct odp_crypto_session_param_t {
 	 *
 	 *  The default value is ODP_CRYPTO_SYNC.
 	 */
-	odp_crypto_op_mode_t pref_mode;
+	odp_crypto_op_mode_t ODP_DEPRECATE(pref_mode);
```

Comment:
compl_queue documentation (under) refers to odp_crypto_operation(). That reference could be removed.

Reply:
I reworded the text but did not completely remove the reference since the usage with odp_crypto_operation() is still part of the deprecated variant of the API.